backend/internal/activities/deployer_activities.go

```diff
@@ -568,6 +568,16 @@ func createDeployerWorkflowTemplate(sse *internal.SSEManager, engine *ewf.Engine
 	return template
 }
 
+func createAddNodeWorkflowTemplate(sse *internal.SSEManager, engine *ewf.Engine, metrics *metrics.Metrics) ewf.WorkflowTemplate {
+	template := newKubecloudWorkflowTemplate()
+	template.AfterWorkflowHooks = append(template.AfterWorkflowHooks,
+		[]ewf.AfterWorkflowHook{
+			notifyWorkflowProgress(sse),
+			CloseClient,
```

Comment:
this will close the client before the verification happens, leading to fail to rollback if node is not verified